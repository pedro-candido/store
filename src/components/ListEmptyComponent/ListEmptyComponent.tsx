import { Image, View } from 'react-native';

function ListEmptyComponent() {
  return <View style={{
    flex: 1, marginHorizontal: 'auto'
  }}><Image width={50} height={50} source={{
    uri: "https://i.gifer.com/ZKZg.gif"
  }} />
  </View>
}

export default ListEmptyComponent