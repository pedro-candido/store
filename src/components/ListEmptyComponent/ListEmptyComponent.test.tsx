import React from 'react';

import { render } from '@testing-library/react-native';
import 'jest-styled-components';

import ListEmptyComponent from './';

describe('ListEmptyComponent', () => {
  it('should match snapshot', () => {
    const testRenderer = render(<ListEmptyComponent />);

    expect(testRenderer).toMatchSnapshot();
  });
});
