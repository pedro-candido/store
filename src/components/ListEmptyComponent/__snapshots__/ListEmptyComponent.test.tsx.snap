// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListEmptyComponent should match snapshot 1`] = `
<View
  style={
    {
      "flex": 1,
      "marginHorizontal": "auto",
    }
  }
>
  <Image
    height={50}
    source={
      {
        "uri": "https://i.gifer.com/ZKZg.gif",
      }
    }
    width={50}
  />
</View>
`;
